src/daemon/http/server.rs

```diff
@@ -116,17 +111,19 @@ pub async fn start_krill_daemon(config: Arc<Config>) -> Result<(), Error> {
     test_data_dirs_or_die(&config);
 
     // Call upgrade, this will only do actual work if needed.
-    pre_start_upgrade(config.clone())?;
+    let upgrade_report = prepare_upgrade_data_migrations(UpgradeMode::PrepareThenFinalise, config.clone()).await?;
+    if let Some(report) = &upgrade_report {
+        finalise_data_migration(report.versions(), config.as_ref())?;
```

Comment:
? Didn't we already finalize the migration by passing `UpgradeMode::PrepareThenFinalize' to `prepare_upgrade_data_migration()` ?

Reply:
renamed it to PrepareToFinalise